### ISSUE TITLE
v_accounts in com.android.browser database

### ISSUE THREAD

**Issue by prmiller91** [chapinb/anparser#3]
![screen shot 2015-01-08 at 9 49 06 am](https://cloud.githubusercontent.com/assets/9575133/5665119/c3bf75e6-971b-11e4-930e-0ba0f46d63e4.png)

The sqlite_plugins.get_sqlite_table_names is not grabbing either v_accounts or v_omnibox_suggestions. Any way around this? I'm assuming there is some kind of conflict with the prepended 'v'.

**Comment by chapinb** [comment]
Is there data within the v_accounts?

**Comment by prmiller91** [comment]
Yes

**Comment by chapinb** [comment]
The code in question is as follows:

```
def get_sqlite_table_names(db_path):

    con = sqlite3.connect(db_path)
    cur = con.cursor()
    cur.execute('Select name from sqlite_master where type = \'table\';')

    tmp =  cur.fetchall()
    tmp2 = []

    for i in tmp:
        for x in i:
            tmp2.append(x)

    return tmp2
```

When running the sql query manually the tables are not displayed.

![2015-01-08_12-01-50](https://cloud.githubusercontent.com/assets/7152035/5666530/2c61608e-972e-11e4-9b2b-4345e12b6c5c.png)

On closer inspection, it appears that v_\* represent views, not tables, and were filtered out because of that. 

![2015-01-08_12-05-40](https://cloud.githubusercontent.com/assets/7152035/5666594/aef3ba92-972e-11e4-8f94-3baa3e936f11.png)

Since they are views, the SQL shown in the screenshots illustrates how we can rebuild those views from existing tables. 

I think we should still grab information from tables only, but should add a new function to read the views and assist the developer in rebuilding them!

I will move this from bug to enhancement

**Comment by chapinb** [comment]
Untested, yet possible solution: https://gist.github.com/chapinb/68a09b9bf4c5c7ccf197

**Comment by prmiller91** [comment]
Here's what I got when printing sqlite_plugins.get_sqlite_view_info(file_path)
![screen shot 2015-01-08 at 8 54 40 pm](https://cloud.githubusercontent.com/assets/9575133/5674940/03f56852-9779-11e4-9e79-e316e4cbc309.png)

I agree though, this isn't a priority issue. If this looks like it'll take too much time to get it to work - let's just put it on the back burner.

**Comment by prmiller91** [comment]
Here's the code that worked:
![screen shot 2015-01-09 at 2 11 11 pm](https://cloud.githubusercontent.com/assets/9575133/5686541/90a3e888-9809-11e4-87ff-40f1aeea3119.png)

It's a bit of a dodgy fix.. but what do you think? It's at least a temporary fix.

Added in bc3bac6

**Comment by prmiller91** [comment]
Actually, I made it too complicated. All you have to do is tmp2.append(i[0]) rather than what I was doing. Fixed in bbb471945673ad76d0b3d27276f2d9898a4f4433
